sui-execution/src/lib.rs

```diff
@@ -25,25 +26,29 @@ pub fn executor(
     protocol_config: &ProtocolConfig,
     paranoid_type_checks: bool,
     silent: bool,
+    enable_profiler: Option<PathBuf>,
 ) -> SuiResult<Arc<dyn Executor + Send + Sync>> {
     let version = protocol_config.execution_version_as_option().unwrap_or(0);
     Ok(match version {
         0 => Arc::new(v0::Executor::new(
             protocol_config,
             paranoid_type_checks,
             silent,
+            enable_profiler,
         )?),
 
         1 => Arc::new(v1::Executor::new(
             protocol_config,
             paranoid_type_checks,
             silent,
+            enable_profiler,
         )?),
 
         2 => Arc::new(latest::Executor::new(
             protocol_config,
             paranoid_type_checks,
             silent,
+            enable_profiler,
```

Comment:
The extra test item in CI says I cant't modify this file? But I need to modify it, is there a suggested workaround?

Reply:
This file is generated by a script, so you need to edit that instead, you can edit this part here:

https://github.com/MystenLabs/sui/blob/437d4857b65fd35517c053018008089427f03f51/scripts/execution_layer.py#L489-L495

And edit the overall template, here:

https://github.com/MystenLabs/sui/blob/main/sui-execution/src/lib.template.rs

To include the imports.  The usual trick for these files is to look at the header comment, which mentions the script that generated the file, and work from there.

Reply:
Do we also run the script or is it done by an automated process?